src/main/pg/displayport_profiles.c

```diff
@@ -30,7 +30,14 @@
 
 #if defined(USE_MSP_DISPLAYPORT)
 
-PG_REGISTER(displayPortProfile_t, displayPortProfileMsp, PG_DISPLAY_PORT_MSP_CONFIG, 1);
+PG_REGISTER_WITH_RESET_FN(displayPortProfile_t, displayPortProfileMsp, PG_DISPLAY_PORT_MSP_CONFIG, 1);
+
+void pgResetFn_displayPortProfileMsp(displayPortProfile_t *displayPortProfile)
+{
+    for (uint8_t font = 0; font < DISPLAYPORT_SEVERITY_COUNT; font++) {
+        displayPortProfile->fontSelection[font] = font;
```

Comment:
So you just want to count up with each array value equal to its index?
E.g. with `DISPLAYPORT_SEVERITY_COUNT = 5`: **0, 1, 2, 3, 4, 0, 0, 0, ...**

I'm asking because I am trying to make sense of its use.

Reply:
This is mapping each of https://github.com/betaflight/betaflight/blob/5d29a0be0656303641b7004edcf8f22c770c5d22/src/main/drivers/display.h#L38 to one of four font pages.

Defaults of `0,1,2,3` result in text with severities of Normal, Info, Warning and Critical being displayed in white, green, amber and red respectively. If you wanted white text for everything except critical warnings, which you want in amber, you’d use `0,0,0,2`.

Reply:
I see, thanks